sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -3015,6 +3059,15 @@ export class BlockBlobClient extends BlobClient {
    * @param {BlockBlobUploadOptions} [options] Options to the Block Blob Upload operation.
    * @returns {Promise<BlockBlobUploadResponse>} Response data for the Block Blob Upload operation.
    * @memberof BlockBlobClient
+   *
+   * @example
+   * ```js
+   * const content = "hello";
+   * const blobName = "newblob" + new Date().getTime();
+   * const blobClient = containerClient.getBlobClient(blobName);
```

Comment:
`containerClient` would be undefined here, is it okay if the example is non-functional?

Reply:
Trying to find a balance between keeping the inline samples elemental enough to be useful but also complete enough to be understandable. I think it will be good to show the construction of containerClient here, so I'll add that in an update.

Reply:
There's also the issue that vanishingly few of these samples are really "functional" or runnable since they use top-level await statements which are not allowed. In order to make them runnable we'd have to wrap all of them in `async function main() { ... }; main();` shims. I'm all for doing that in the README but I'm hesitant to add that much to the inline examples. Either way we should figure out whether we want these examples to be really elemental or if they should be copy-pasteable to run in Node.